services/tenant-ui/frontend/src/store/stateStore.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Place for storing global state that is not persisted in the backend.
+ */
+import { defineStore } from 'pinia';
+import { ref } from 'vue';
+
+export const useGlobalStateStore = defineStore('globalState', () => {
```

Comment:
All the stores are global and represent state so maybe a different naming here? "Common" or "Layout" or something?

Reply:
I like your thinking here.